learn/guides/asyncapi-tool.md

```diff
@@ -43,117 +129,37 @@ The generated Ballerina sources will be written into the same directory from whi
-              type: string
-          required:
-            - type
-            - event_ts
-          title: "The actual event, an object, that happened"
-          type: object
-      required:
-        - event
-      title: Standard event wrapper for the Events API
-      type: object
-channels:
-  app:
-    subscribe:
-      message:
-        oneOf:
-          - x-ballerina-event-type: "app_mention"
-            externalDocs:
-              description: Event documentation for app_mention
-            payload:
-              $ref: "#/components/schemas/GenericEventWrapper"
-            summary: Subscribe to only the message events that mention your app or bot
-            tags:
-              - name: allows_user_tokens
-              - name: app_event
-            x-scopes-required: []
-            x-tokens-allowed:
-              - user
-```
-
-There are custom tags in this YAML starting with `x-ballerina`. Those tags are being used for the purposes below.
-
-- `x-ballerina-event-identifier` - when the listener receives an event from the event source (Slack is the event source in this scenario), there should be a way to identify the event type. This includes two parts, `type` and `path`. Type can be either `header` or `body`. In other words, the type of the event can be included in the payload either as a header or as an attribute in the body.
-
-  > **Note:** Currently, this tool supports only HTTP-based event APIs. The path is equal to the header name if the type is `header`, or to the JSON path of the attribute if the type is `body`.
-
-- `x-ballerina-event-type` - this should be there in every event inside the channel. This is the name of the event or the value of the above-mentioned attribute for a specific event.
-
-### Generate Ballerina sources
-
-Execute the command below on the above file. This will generate four files as mentioned above.
-
-```bash
-bal asyncapi -i slack.yaml
-```
-
-### Modify the generated sources
-
-The generated files can be modified according to the custom requirements. When modifying the generated code segments, it will be easier to consider the facts below.
+Then, the generated files can be modified according to the custom requirements. When modifying the generated code segments, it will be easier to consider the below facts.
 
 - All the incoming requests will be coming to the resource function in the `dispatcher_service.bal` file. Hence, if there is a necessity to add an authentication logic for the incoming calls, that logic can be included there before processing the incoming HTTP request.
-- If more information is needed when initializing the listener like secrets, endpoint URLs, tokens, refresh tokens, etc., update the `init` function in the `listener.bal` file.
-
-### Execute the generated sources
+- If more information is needed when initializing the listener like secrets, endpoint URLs, tokens, refresh tokens, and so on, update the `init` function in the `listener.bal` file.
```

Comment:
```suggestion
- If more information is needed when initializing the listener such as secrets, endpoint URLs, tokens, refresh tokens, etc., update the `init` function in the `listener.bal` file.
```

Reply:
Thanks a lot for the suggestion, Resolved